design/tokens/components/iconButton/theme/iconButton.sizes.ts

```diff
@@ -0,0 +1,21 @@
+export default {
+  lg: {
+    apply: 'utility-styles-icon-button-large',
+    height: '2rem',
+    width: '2rem',
+    padding: '0.5rem',
+    borderRadius: '0.5rem'
+  },
+  md: {
+    apply: 'utility-styles-icon-button-base',
+    height: '1.5rem',
+    width: '1.5rem',
+    padding: '0.25rem',
+  },
+  sm: {
+    apply: 'utility-styles-icon-button-small',
+    height: '1.25rem',
+    width: '1.25rem',
+    padding: '0.25rem'
+  }
```

Comment:
![image](https://github.com/decentdao/decent-ui/assets/38386583/4b316ae3-9db2-4083-b058-13309ba72c80)

The sm icon seems like the padding is off. This was tricky with the IconButton as the width and height properties are for the `Button` with the SVG just being relative to that.